dataloader/segments_parallels.py

```diff
@@ -142,7 +147,71 @@ def load_segments(segments: list, all_parallels: list, db: StandardDatabase) ->
 
     return segmentnrs, totallengthcount, totalfilelengthcount
 
-
+def load_search_index(path,path_chn, db: StandardDatabase):
+    with gzip.open(path) as f:
+        print(f"\nLoading file index data...")
+        index_data = json.load(f)
+        print(f"\nInserting file index data into DB...")
+        collection = db.collection(COLLECTION_SEARCH_INDEX)
+        # we have to do this in chunk, otherwise it will fail with broken_pipe
+        chunksize = 10000
+        for i in tqdm(range(0,len(index_data),chunksize)):
+            collection.insert_many(index_data[i:i+chunksize])
+        print(f"\nDone loading index data...")
+        print(f"\nCreating View...")
+        db.create_arangosearch_view(
+            name=VIEW_SEARCH_INDEX,
+            properties={'cleanupIntervalStep': 0, "links" : { 
```

Comment:
can we import this `properties` key from some other file to keep this function short and readable?